src/buildercore/terraform.py

```diff
@@ -63,6 +66,8 @@
 # https://github.com/terraform-providers/terraform-provider-fastly/issues/7 tracks when snippets could become available in Terraform
 FASTLY_MAIN_VCL_KEY = 'main'
 
+# e.g. 404: 'Not Found'
+FASTLY_RESPONSE_REASON_PHRASES = http_responses()
```

Comment:
Is `FASTLY_RESPONSE_REASON_PHRASES` really needed?

Reply:
It's a little abstraction layer but I'll put a docstring on the original function instead